### PR TITLE
Docs: add list of game interface elements that we want to capitalize …

### PR DESCRIPTION
**Pull request by jwrober**
…to style guide

### CODE REVIEW

**Review by daavko — COMMENTED** [review]
I don't think we should be listing prefixes here. Things like `Special (also covers Specialist)`, `Entertain(er)`, `Prod(uction)`. IMO this should be just a plain list of terms that are capitalized, without explicitly listed forms of the word.

**Comment by jwrober** [comment]
> I don't think we should be listing prefixes here. Things like `Special (also covers Specialist)`, `Entertain(er)`, `Prod(uction)`. IMO this should be just a plain list of terms that are capitalized, without explicitly listed forms of the word.

I did the prefixes with notes to aid another person in using find/replace. That is the method I used while updating `helpdata.txt` and the `tutorial.sav` scenario with the new capitalization rules. Do you have another suggestion?

**Comment by daavko** [comment]
I'd suggest making this just a list of in-game terms that are capitalized, organized by category, since it's just a style guide. There could, of course, be a "useful list for find/replace for those terms", but I'd put that on a separate page. Or a collapsible section under the main list, if that can be done.

File: docs/Contributing/style-guide.rst [hunk 1]
```diff
@@ -156,6 +156,113 @@ Capitalization
     This page provides a good overview of Title Case capitalization rules for US English:
     https://www.grammarly.com/blog/capitalization-rules/
 
+    Below is a more exhaustive list of game specific terms that we want to capitalize. The list can be used
+    in a find/replace exercise:
+
+    :strong:`Game Interace`:
```
Comment (lmoureaux) [inline docs/Contributing/style-guide.rst:162]: Interface

File: docs/Contributing/style-guide.rst [hunk 2]
```diff
@@ -156,6 +156,113 @@ Capitalization
...
+      * Advance (as in Technology Advance)
+      * Airbase
+      * Base
+      * Border
+      * Bulbs
+      * Buoy
+      * Button (a named button)
+      * Chat
+      * Cit(y or ies)
```
Comment (lmoureaux) [inline docs/Contributing/style-guide.rst:171]: City/Cities

File: docs/Contributing/style-guide.rst [hunk 3]
```diff
@@ -156,6 +156,113 @@ Capitalization
...
+      * Dialog (a named dialog)
+      * Embass(y or ies)
```
Comment (lmoureaux) [inline docs/Contributing/style-guide.rst:173]: Embassy/Embassies

File: docs/Contributing/style-guide.rst [hunk 4]
```diff
@@ -156,6 +156,113 @@ Capitalization
...
+      * Fort(ress)
```
Comment (lmoureaux) [inline docs/Contributing/style-guide.rst:174]: Fort and Fortress

File: docs/Contributing/style-guide.rst [hunk 5]
```diff
@@ -156,6 +156,113 @@ Capitalization
...
+      * Improvement
+      * Map
+      * Menu (a named menu)
+      * Messages
+      * Minimap
+      * Small Great Wonder (only with Wonders)
+      * Tax(es) Rate(s)
```
Comment (lmoureaux) [inline docs/Contributing/style-guide.rst:181]: "Taxes Rates", really? Maybe this should be two items

**Review by lmoureaux — COMMENTED** [review]
I agree with Daavko that it would be more clear to skip the prefix thing entirely and just list the variants when it makes sense. I was actually confused the first time and it's probably going to confuse non-programmers even more (we're used to this kind of reasoning from regexes).

Some of these terms, like Prod, may sometimes be abbreviated. In such a case the abbreviation could be listed next to it.

I think this would be better rendered as one or more [hlists](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html?highlight=hlists#directive-hlist) with subtitles inbetween (HTML gives us 6 heading levels, let's not be shy on using them).

**Pull request merged**